src/main/resources/com/google/api/codegen/ruby/test.snip

```diff
@@ -0,0 +1,124 @@
+@extends "ruby/method_sample.snip"
+
+@snippet generate(testViewModel)
+  # TODO: Use testViewModel.fileHeader here and in main.snip.
+  @# Copyright 2016 Google Inc. All rights reserved.
+  @#
+  @# Licensed under the Apache License, Version 2.0 (the "License");
+  @# you may not use this file except in compliance with the License.
+  @# You may obtain a copy of the License at
+  @#
+  @#     http://www.apache.org/licenses/LICENSE-2.0
+  @#
+  @# Unless required by applicable law or agreed to in writing, software
+  @# distributed under the License is distributed on an "AS IS" BASIS,
+  @# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  @# See the License for the specific language governing permissions and
+  @# limitations under the License.
+
+  require "minitest/autorun"
+  require "{@testViewModel.libraryUnderTest.outputFile}"
+  include {@testViewModel.libraryUnderTest.namespace}
```

Comment:
Is there a reason why this `include` is global, rather than in the `describe` block?

This is a bit of a code smell.  Using `include` at all to include members into the current scope for convenience is also a bit non-idiomatic.

If you want to not have to use fully-qualified class names in the test, have you considered using a constant alias rather than include?

```ruby
describe {@testViewModel.libraryUnderTest.apiClassName} do
  {@testViewModel.libraryUnderTest.apiClassName} = {@testViewModel.libraryUnderTest.namespace}
```

If those are the right template properties... inotherwords something that generates like:

```ruby
describe {@testViewModel.libraryUnderTest.apiClassName} do
  Language = Google::Cloud::Language::V1beta1

  # ...
  Language::LanguageService::Stub.stub(:new, mock_client) { yield(mock_client) }
```

This is similar to what we do in code samples using the libraries.